nautobot/ui/src/components/ObjectTable/ObjectListTable.js

```diff
@@ -187,16 +189,13 @@ export default function ObjectListTable({
                     </Heading>
                     <Spacer />
                     {!data_fetched ? (
-                        <Box pr="sm">
+                        <Box marginRight="md">
                             <LoadingWidget name={tableTitle} />
                         </Box>
-                    ) : (
-                        () => {}
-                    )}
+                    ) : null}
```

Comment:
```javascript
{!data_fetched && (
         <Box marginRight="md">
             <LoadingWidget name={tableTitle} />
         </Box>
   )}
```
I guess no need of the `null`

Reply:
It certainly is possible and well supported by React, though for safety and consistency the community prefers to use ternary operators anyway. Here's a blog post on that subject by Kent C. Dodds: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx. I really recommend reading his blog as well as watching his talks, most of the time his thought process is brilliant.